config/mf-strict.conf

```diff
@@ -26,6 +26,9 @@ name: "Micro Focus COBOL"
 # Value: enum (see default.conf and cobc/cobc.h for details)
 standard-define			1
 
+# Default source reference-format (see default.conf for details)
+format:				fixed # CHECKME
```

Comment:
checked, that's definitive the case, 01 declarations and section names can start in column 13 and code in column 8.